config/crossSections.yaml

```diff
@@ -0,0 +1,748 @@
+  unc: 0.03993
+ggX_HH_bbtautau:
+  crossSec: 1
+  reference:
+  unc:
+VBFX_HH_bbtautau:
+  crossSec: 1
+  reference:
+  unc:
+ggHH_bbTauTau_SM:
+  crossSec: 31.05*0.001 *0.5824*0.06272
+  reference: Higgs_XSBR_YR4_update https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageAt13TeV
+  unc:
+GluGluToHHTo2B2Tau_node:
+  crossSec: 1
+  reference:
+  unc:
+TTZToBB:
+  crossSec: 0.1118
+  reference: XSDB TTZToBB_TuneCP5_13TeV-amcatnlo-pythia8 modifiedOn 2020-10-10 11:41:16
+  unc: 0.0001631
+TTZToNuNu:
+  crossSec: 0.1476
+  reference: XSDB TTZToNuNu_TuneCP5_13TeV-amcatnlo-pythia8 modifiedOn 2021-06-21 09:58:16
+  unc: 0.0001971
+TTZToQQ_Dilept:
+  crossSec: 0.0568
+  reference: XSDB TTZToQQ_Dilept_TuneCP5_13TeV-amcatnlo-pythia8 modifiedOn 2021-06-21 09:58:18
+  unc: 8.059e-05
+TTGJets:
+  crossSec: 3.757
+  reference: XSDB TTGJets_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8 modifiedOn 2021-06-21 09:55:33
+  unc: 0.02243
+TTGamma_Dilept:
+  crossSec: 1.494
+  reference: XSDB TTGamma_Dilept_TuneCP5_13TeV-madgraph-pythia8 modifiedOn 2021-06-21 09:55:35
+  unc: 0.001264
+TTGamma_Hadronic:
+  crossSec: 4.178
+  reference: XSDB TTGamma_Hadronic_TuneCP5_13TeV-madgraph-pythia8 modifiedOn 2021-06-21 09:55:40
+  unc: 0.003734
+TTGamma_SingleLept:
+  crossSec: 5.095
+  reference: XSDB TTGamma_SingleLept_TuneCP5_13TeV-madgraph-pythia8 modifiedOn 2021-06-21 09:55:44
+  unc: 0.00342
+TTTJ:
+  crossSec: 0.0003974
+  reference: XSDB WZTo1L1Nu2Q_13TeV_TuneCP5_amcatnloFXFX_madspin_pythia8 modifiedOn 2018-07-10 12:48:41
+  unc: 2.299e-7
+TTTT:
+  crossSec: 0.008213
+  reference: XSDB TTTT_TuneCP5_13TeV-amcatnlo-pythia8 modifiedOn 2018-07-10 12:48:42
+  unc: 1.189e-05
+TTTW:
+  crossSec: 0.0007314
+  reference: XSDB TTTW_TuneCP5_13TeV-madgraph-pythia8 modifiedOn 2018-07-10 12:48:43
+  unc: 4.889e-07
+TTZZTo4b:
+  crossSec: 0.001387
+  reference: XSDB TTZZTo4b_TuneCP5_13TeV-madgraph-pythia8 modifiedOn 2022-03-02 06:43:02
+  unc: 1..006e-06
+TTZZToNon4b:
+  crossSec: 0.00139
+  reference: XSDB TTZZToNon4b_TuneCP5_13TeV-madgraph-pythia8 modifiedOn 2022-03-02 06:43:03
+  unc: 1.006e-06
+TT4b:
+  crossSec: 0.06683
+  reference: XSDB TT4b_TuneCP5_13TeV_madgraph_pythia8 2020-09-24 05:12:25
+  unc: 5.292e-05
+TTZHTo4b:
+  crossSec: 0.001135
+  reference: XSDB TTZHTo4b_TuneCP5_13TeV-madgraph-pythia8 modifiedOn 2022-03-02 06:42:54
+  unc: 7.972e-07
+TTZHToNon4b:
+  crossSec: 0.001134
+  reference: XSDB TTZHToNon4b_TuneCP5_13TeV-madgraph-pythia8 modifiedOn 2022-03-02 06:43:02
+  unc: 7.889e-07
+TTHHTo4b:
+  crossSec: 0.0006676
+  reference: XSDB TTHHTo4b_TuneCP5_13TeV-madgraph-pythia8 modifiedOn 2022-03-02 06:42:48
+  unc: 4.633e-06
+TTHHToNon4b:
+  crossSec: 0.0006679
+  reference: XSDB TTHHToNon4b_TuneCP5_13TeV-madgraph-pythia8 modifiedOn 2022-03-02 06:42:49
+  unc: 4.628e-07
+TTHH:
+  crossSec: 0.0006655
+  reference: XSDB TTHH_TuneCP5_13TeV-madgraph-pythia8 modifiedOn 2018-07-10 12:48:39
+  unc: 5.345e-07
```

Comment:
take values from twiki H or HH twiki or from YR

Reply:
I did not find these in the twikis